README.md

```diff
@@ -94,7 +94,7 @@ Watson services are migrating to token-based Identity and Access Management (IAM
 
 - With some service instances, you authenticate to the API by using **[IAM](#iam)**.
 - In other instances, you authenticate by providing the **[username and password](#username-and-password)** for the service instance.
-- If you're using a Watson service on ICP, you'll need to authenticate in [a specific way](icp).
+- If you're using a Watson service on ICP, you'll need to authenticate in [a specific way](#ICP).
```

Comment:
I think this needs to be lowercase, not showing up as a link in the rich diff
```suggestion
- If you're using a Watson service on ICP, you'll need to authenticate in [a specific way](#icp).
```

Reply:
Updated